checkov/github_actions/runner.py

```diff
@@ -181,13 +186,22 @@ def resolve_step_name(job_definition: dict[str, Any], start_line: int, end_line:
     @staticmethod
     def is_schema_valid(config: dict[str, Any] | list[dict[str, Any]]) -> bool:
         valid = False
-        config = force_list(config)
+        config = force_dict(config)
+        dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
```suggestion
        dir_path = os.path.dirname(__file__)
```
please never use `realpath` it has unwanted behaviour on Mac for example